pep-0637.rst

```diff
@@ -807,7 +807,15 @@ This proposal already established that, in case no positional index is given, th
 passed value must be the empty tuple. Allowing for the empty index notation would
 make the dictionary type accept it automatically, to insert or refer to the value with
 the empty tuple as key. Moreover, a typing notation such as ``Tuple[]`` can easily
-be written as ``Tuple`` without the indexing notation.
+be written as ``Tuple`` without the indexing notation. 
+
+However, subsequent discussion with Brandt Bucher during implementation has revealed
+that the case ``obj[]`` would fit a natural evolution for default generics, giving 
```

Comment:
My points were more directed towards variadic generics (I suppose "default" generics could be a possible use, but I haven't really given much thought to that).

```suggestion
that the case ``obj[]`` would fit a natural evolution for variadic generics, giving 
```

I take it Steven didn't budge on this? :)